engine/src/main/java/org/hibernate/search/engine/service/impl/StandardServiceManager.java

```diff
@@ -58,6 +62,7 @@ public StandardServiceManager(SearchConfiguration searchConfiguration,
 		this.providedServices = createProvidedServices( searchConfiguration );
 		this.defaultServices = defaultServices;
 		this.classloaderService = searchConfiguration.getClassLoaderService();
+		this.failOnUnreleasedService = Boolean.parseBoolean( System.getProperty( "org.hibernate.search.fail_on_unreleased_service" ) );
```

Comment:
there's a short version for that:

    Boolean.getBoolean( "org.hibernate.search.fail_on_unreleased_service" );